network/src/commonMain/kotlin/com/wire/kalium/network/api/asset/AssetApiImpl.kt

```diff
@@ -74,6 +77,11 @@ class AssetApiImpl internal constructor(
             }
         }
 
+    override suspend fun deleteAsset(assetId: AssetId, assetToken: String?): NetworkResponse<Unit> =
```

Comment:
Would be nice if we add a test for this case 🤾

Reply:
I will add it in next PR with local asset removal 💪